crates/sui-indexer-alt-consistent-store/src/db/mod.rs

```diff
@@ -536,6 +637,42 @@ impl From<CommitterWatermark> for Watermark {
     }
 }
 
+/// Retrieves a RocksDB property from db and maps it to a metric value.
+fn cf_property_int_to_metric(
+    db: &rocksdb::DB,
+    cf: &impl AsColumnFamilyRef,
+    property_name: &std::ffi::CStr,
+) -> i64 {
+    match db.property_int_value_cf(cf, property_name) {
+        Ok(Some(value)) => Ok(value.min(i64::MAX as u64).try_into().unwrap_or_default()),
+        Ok(None) => Ok(0),
+        Err(e) => Err(e.into_string()),
+    }
+    .unwrap_or(METRICS_ERROR)
```

Comment:
There's an inconsistency here -- if the column family does not exist, we return `-1` for the metric, but if the metric does not exist, we return `0` -- suggest unifying these to both `-1`:


```suggestion
        Ok(None) | Err(_) => METRICS_ERROR,
    }
```